lib/generators/rolemodel/lograge/templates/config/initializers/lograge.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+unless %w[development test].include?(Rails.env)
+  Rails.application.configure do
+    config.lograge.enabled = true
+    config.lograge.custom_options = lambda do |event|
+      exceptions = %w[controller action format id]
+      {
+        params: event.payload[:params].except(*exceptions)
```

Comment:
Braden and I talked about adding `user_id` here too. Could we do this?

```suggestion
        params: event.payload[:params].except(*exceptions),
        # user_id: controller.current_user.try(:id)
```